libs/barista-components/slider/barista.json

```diff
@@ -0,0 +1,24 @@
+{
+  "title": "Slider",
+  "description": "Component that mimics the behavior of a html range node.",
```

Comment:
I would write a description that is more UX focused instead of developer-focused. This should explain which use case it would solve.

Reply:
I'll try to come up with a better one :)